### DISCUSSION TITLE
Building a (emu) SPDM-Implementation on two Nvidia Jetson Orins

### DISCUSSION THREAD

**Discussion by marinuswhite** [DMTF/libspdm#2660] [Q&A] [answered]
Hello everyone, After succesfully testing the SPDM emulation on one Nvidia Jetson Orin, I would now like to extend this setup to two Orins. One Orin would serve as Responder and the other as the Requester. For the Beginning Ethernet is to be used as the SMBus. Therefore, my question is whetever I can simply split the emulation between the two devices (tell the emulation to send the packages via Ethernet and not internaly if that's possible?), or is a complete Responder/Requester implementation necessary in this case? If a complete implementation is required, are there any existing solutions available, or would the best approach be to develop this myself (via [libspdm/doc/api/responder_api.md at main · DMTF/libspdm (github.com)](https://github.com/DMTF/libspdm/blob/main/doc/api/responder_api.md)) if I unterstood that correctly? Thank you in advance!

**Comment by steven-bellock** [comment, marked as answer]
Have a look at https://github.com/westerndigitalcorporation/spdm-utils. If that doesn't work then yes, you could extend `spdm-emu` or just make your own Requester and/or Responder using libspdm.
